NEWS

```diff
@@ -1,3 +1,21 @@
+Overview of changes in 0.0.90
+============================
+
+* Rewrite Toolbox in Go
+* Add option --log-podman to regulate output of Podman
+* Add option --version to show current Toolbox version
+* Fix duplication in the output of the list command
+* Make option --container in commands create and enter an argument
+* Make bash completion a bit more reliable
+* Make the create command abortable
+* Fix leaking of spinners when aborted
+* Replace option --verbose and --very-verbose with --log-level while keeping
+  verbose for backwards compatibility
+* Mark the reset command as deprecated (replaced by 'podman system reset')
+* Drop some no longer needed code
```

Comment:
We should be more explicit by saying something like: *we dropped compatibility with ...*.

Reply:
Yes, true. I did not understand some of those code paths, so I was a bit too general.  The line could be dropped completely.

Reply:
We should still mention it though because it might still affect some users. Who knows? We just need to be more explicit about it.